src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -140,6 +140,9 @@ $tooltip-color: $white !default;
 $tooltip-bg: $body-color !default;
 $tooltip-opacity: 1 !default;
 
+// Side panel
+$side-panel-overlay-bg: rgba(0, 0, 0, 0.4) !default;
```

Comment:
Shouldn't this be taken from one of carbon's color tokens ?https://carbondesignsystem.com/elements/color/overview/#themes

Also, in the `_carbon-defaults.scss`?

Reply:
it looks like `_bootstrap-defaults` is not going anywhere soon. infact _carbon-default colors are actually overriden using the colors in this file. So kept the variable there but imported the `$overlay` token inside it.